perf/benchmark/runner/fortio.py

```diff
@@ -243,7 +243,7 @@ def getParser():
     parser.add_argument(
         "--table", help="Name of the BigQuery table to send results to, like istio_perf_01.perf", default=None)
     parser.add_argument("--selector", help="timestamps to match for import")
-    parser.add_argument("--csv", help="columns in the csv file", default="StartTime,ActualDuration,Labels,NumThreads,ActualQPS,p50,p99,cpu_mili_avg_telemetry_mixer,cpu_mili_max_telemetry_mixer,mem_MB_max_telemetry_mixer,cpu_mili_avg_fortioserver_deployment_proxy,cpu_mili_max_fortioserver_deployment_proxy,mem_MB_max_fortioserver_deployment_proxy,cpu_mili_avg_ingressgateway_proxy,cpu_mili_max_ingressgateway_proxy,mem_MB_max_ingressgateway_proxy")
+    parser.add_argument("--csv", help="columns in the csv file", default="StartTime,ActualDuration,Labels,NumThreads,ActualQPS,p50,p90,p99,cpu_mili_avg_telemetry_mixer,cpu_mili_max_telemetry_mixer,mem_MB_max_telemetry_mixer,cpu_mili_avg_fortioserver_deployment_proxy,cpu_mili_max_fortioserver_deployment_proxy,mem_MB_max_fortioserver_deployment_proxy,cpu_mili_avg_ingressgateway_proxy,cpu_mili_max_ingressgateway_proxy,mem_MB_max_ingressgateway_proxy")
```

Comment:
I think we still have the full flag in the readme, maybe want to update it now that we have default, unless we already did